packages/ag-charts-community/src/chart/series/cartesian/barSeries.ts

```diff
@@ -539,24 +540,26 @@ export class BarSeries extends AbstractBarSeries<
         if (processedData.type === 'grouped') {
             const width = barWidth;
 
-            const yStartValues = dataModel.resolveColumnById(this, `yValue-start`, processedData);
-            const yEndValues = dataModel.resolveColumnById(this, `yValue-end`, processedData);
-            const yRangeIndex = dataModel.resolveProcessedDataIndexById(this, `yValue-range`);
+            const stacked = dataModel.hasColumnById(this, `yValue-start`);
+            const yStartValues = stacked ? dataModel.resolveColumnById(this, `yValue-start`, processedData) : [];
+            const yEndValues = stacked ? dataModel.resolveColumnById(this, `yValue-end`, processedData) : [];
+            const yRangeIndex = stacked ? dataModel.resolveProcessedDataIndexById(this, `yValue-range`) : -1;
```

Comment:
This helps ensuring you don't forget to do the checks later:-

```suggestion
            const yStartValues = stacked ? dataModel.resolveColumnById(this, `yValue-start`, processedData) : undefined;
            const yEndValues = stacked ? dataModel.resolveColumnById(this, `yValue-end`, processedData) : undefined;
            const yRangeIndex = stacked ? dataModel.resolveProcessedDataIndexById(this, `yValue-range`) : undefined;
```